addons/web_editor/static/src/js/wysiwyg/widgets/link_popover_widget.js

```diff
@@ -140,6 +140,20 @@ const LinkPopoverWidget = Widget.extend({
                 this.$target.popover('update');
             });
         }
+        this._showLinkTools();
```

Comment:
If I get this right, the purpose of this line is to say "When I click on a link, show the right panel link tool", right ?

**If** this is the case, this is not the 'job' of the popover widget. It is quite the same explanation than https://github.com/odoo/odoo/pull/75205#discussion_r698462171 for the double click.
If we remove the popover widget one day (trust me we will), we don't want to lose this behavior, as we will still want click on links to open the right panel link tool.

Reply:
@rdeodoo Moved outside of the popover